### DISCUSSION TITLE
Jobstores

### DISCUSSION THREAD

**Discussion by Shukhratbek-Kholmatov** [agronholm/apscheduler#747] [Q&A]
Hello everyone,I configure AsynIoscheduler to Sqlaalchemy jobstores,i run apscheduler in pycharm and jobs append in sqlalchemy jobstores(sqlite db).But when i deploy my script on a server jobs sometimes don't append.Please help me

**Comment by agronholm** [comment]
You really need to elaborate on that. You're essentially saying "SQLAlchemy job store doesn't work for me". That's not helpful. Show some code, or provide debug logging at the very least. Preferably both. Especially in cases where things "sometimes" don't work.

**Reply by Shukhratbek-Kholmatov** [reply]
![photo_1_2023-06-20_19-10-53](https://github.com/agronholm/apscheduler/assets/83421087/ca370edc-aaf6-4585-9bc4-05a800497217)
![photo_2_2023-06-20_19-10-53](https://github.com/agronholm/apscheduler/assets/83421087/aa5b4559-372f-4862-99d8-49e4f32e6252)

**Reply by Shukhratbek-Kholmatov** [reply]
these loader and app files my code
